devfile.yaml

```diff
@@ -2,15 +2,46 @@ schemaVersion: 2.0.0
 metadata:
   name: nodejs
   version: 1.0.0
-  alpha.build-dockerfile: https://raw.githubusercontent.com/odo-devfiles/registry/master/devfiles/nodejs/build/Dockerfile
-  alpha.deployment-manifest: https://raw.githubusercontent.com/odo-devfiles/registry/master/devfiles/nodejs/deploy/deployment-manifest.yaml
+  attributes:
+    alpha.build-dockerfile: https://raw.githubusercontent.com/odo-devfiles/registry/master/devfiles/nodejs/build/Dockerfile
+    alpha.deployment-manifest: https://raw.githubusercontent.com/odo-devfiles/registry/master/devfiles/nodejs/deploy/deployment-manifest.yaml
 starterProjects:
 - name: nodejs-starter
   git:
     remotes:
       origin: https://github.com/odo-devfiles/nodejs-ex.git
 components:
 - name: runtime
+  attributes:
+    first: a
```

Comment:
we have more meaningful name if we want it to be a test devfile in the public repo

Reply:
updated